actionpack/test/dispatch/cookies_test.rb

```diff
@@ -408,6 +418,30 @@ def test_setting_cookie_with_strict
     assert_equal({ "user_name" => "david" }, @response.cookies)
   end
 
+  def test_setting_cookie_with_false
+    @request.env["action_dispatch.cookies_same_site_protection"] = proc { false }
+
+    get :authenticate
+    assert_cookie_header "user_name=david; path=/"
+    assert_equal({ "user_name" => "david" }, @response.cookies)
+  end
+
+  def test_setting_cookie_with_specific_same_site_strict
+    @request.env["action_dispatch.cookies_same_site_protection"] = proc { :lax }
```

Comment:
This is already [set in `setup`](https://github.com/rails/rails/blob/31b1403919eec0973921ab42251bfbbf3d4422b6/actionpack/test/dispatch/cookies_test.rb#L361), so we could possibly omit.  I do see an argument for being explicit here, though.

Reply:
I think it is better to be explicit since all the tests around it have the same pattern.